setup.py

```diff
@@ -57,6 +71,30 @@ def is_requirement(line):
     )
 
 
+def get_edx_platform_version():
+    """
+    Returns the pip package version for Open-edX.
+    """
+    FILE_NAME = 'open_edx_version.txt'
+    VERSION_KEY_NAME = 'Version'
```

Comment:
Isn't better to register these constant as global, at the top of the file? @andrey-canon

Reply:
I haven't found the best practices for the constants definition, but the local definition  uses less memory.

Reply:
Best practice, at module level: https://www.python.org/dev/peps/pep-0008/#constants
Could you share please the link where you found that the local variables use less memory than global? @andrey-canon

Reply:
sorry, less time in memory https://stackoverflow.com/questions/34720384/python-most-convenient-way-to-define-constant-variables-used-just-once

Reply:
Ok, since they are only used in this function.